drake/examples/Pendulum/test/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+
+include_directories(${PROJECT_SOURCE_DIR}/core)
+include_directories(${PROJECT_SOURCE_DIR}/systems)
+include_directories(${PROJECT_SOURCE_DIR}/systems/frames)
+include_directories(${PROJECT_SOURCE_DIR}/systems/plants)
+include_directories(${PROJECT_SOURCE_DIR}/systems/plants/joints)
+include_directories(${PROJECT_SOURCE_DIR}/systems/controllers)
+include_directories(${PROJECT_SOURCE_DIR}/util)
+
+if (LCM_FOUND AND snopt_c_FOUND)
+  add_executable(pendulumURDFDynamicsTest urdfDynamicsTest.cpp)
```

Comment:
i totally agree.  this is a serious discussion to have.  the reason is because RigidBodySystem uses snopt to resolve the initial constraints for systems that have position constraints.  and because the pendulum knows how to publish itself on lcm.  but I'm worried about the dependence creeping that's happening here.